examples/getstarted/config/admin.js

```diff
@@ -22,7 +22,7 @@ module.exports = ({ env }) => ({
     enabled: env.bool('PREVIEW_ENABLED', true),
```

Comment:
Unrelated but I'm noticing this exists in addition to the feature check. We'll need to clarify how we want to do these things